README.md

```diff
@@ -22,30 +24,84 @@ To use it in Maven, add the following to your POM:
 ```
 [//]: # ({x-version-update-end})
 
-Dependency Dashboard
-------------
+When you use the Libraries BOM, you don't specify individual library
+versions and your application runs on a consistent set of the dependencies.
+
+## Libraries in Scope
+
+The content of the Libraries BOM consists of 2 categories: Google Cloud Java
+client libraries and core Google dependency libraries.
+
+### Google Cloud Java client libraries
+
+A Google Cloud Java client library is a Java library that communicates with a
+corresponding GCP service.
+We have ~150 Java client libraries for Google Cloud.
+
+The `google-cloud-bom` module in this repository produces a BOM that lists
+the consistent versions of the Java client libraries. This consistency means
+that these versions are built and tested with the same set of dependencies.
+The Libraries BOM imports `google-cloud-bom`.
+
+### Core Google dependency libraries
+
+Core Google dependency libraries are the Google-developed Java libraries used by
+the client libraries.
+This includes low-level libraries such as Guava, Protobuf, gRPC, Google Auth
+Libraries.
+These libraries do not have a specific Google Cloud service
+associated to it.
+
+While the users of Google Cloud do not have to directly interact with the core
```

Comment:
I'm not sure that strictly accurate. They may interact with Guava and Auth classes, for example.

Reply:
Updated to clarify the meaning.